tests/getUserConfig-test.js

```diff
@@ -3,21 +3,23 @@ import expect from 'expect'
 import getUserConfig from '../src/getUserConfig'
 
 describe('getUserConfig()', () => {
-  describe('when no config file can be found', () => {
-    it('throws an error', () => {
-      expect(getUserConfig)
-        .withArgs({config: 'tests/fixtures/nonexistent.js'})
-        .toThrow(/couldn't find a config file/)
-    })
-  })
+  // TODO: Remove
```

Comment:
Rather than removing, why not test whether you get the expected default configuration?